pyteal/ast/opup.py

```diff
@@ -29,14 +29,14 @@ class OpUpMode(Enum):
     OnCall = 1
 
 
-ON_CALL_APP = Bytes("base16", "068101")  # v6 teal program "int 1"
+ON_CALL_APP = Bytes("base16", "068101")  # v6 avm program "int 1"
```

Comment:
We should probably retain "teal" here, since it describes the opcode language

Reply:
> Our terminology around that isn't changing, is it?

No - terminology not changing here.  Another alternative is to write `v6 program`.